pytket/extensions/aqt/multi_zone_architecture/circuit/multizone_circuit.py

```diff
@@ -366,6 +380,26 @@ def copy(self) -> "MultiZoneCircuit":
         new_circuit.multi_zone_operations = deepcopy(self.multi_zone_operations)
         return new_circuit
 
+    def get_n_shuttles(self) -> int:
+        count = 0
+        for cmd in self.pytket_circuit.get_commands():
+            op = cmd.op
+            optype = op.type
+            op_string = f"{op}"
+            if optype == OpType.CustomGate and "SHUTTLE" in op_string:
+                count += 1
+        return count
+
+    def get_n_pswaps(self) -> int:
```

Comment:
See above

Reply:
Just there for characterization of circuit after compilation. I added some test code for it.